packages/webhooks/tasks/sendModelWebhooks.js

```diff
@@ -17,5 +17,5 @@ async function sendModelWebhooks (modelName) {
 }
 
 module.exports = {
-    sendModelWebhooks: createTask('sendModelWebhooks', sendModelWebhooks, { priority: 3 }),
-}
\ No newline at end of file
+    sendModelWebhooks: createTask('sendModelWebhooks', sendModelWebhooks),
```

Comment:
Please add `low` here. Webhooks are spamming tasks and not that important..,

Reply:
Good suggestion, I'll do it